checklist.html.md.erb

```diff
@@ -26,9 +26,11 @@ Review each of the following links to understand the changes in the new release,
 	* [Ops Manager v2.3 Known Issues](../../pcf-release-notes/opsmanager-rn.html#known-issues)
 	* [PAS v2.3 Known Issues](../../pcf-release-notes/runtime-rn.html)
 	* The following Pivotal Knowledge Base (KB) articles describe known issues for **PCF v2.3**:
-		* [PAS backup-prepare job renamed to backup-restore may cause automation failures](https://community.pivotal.io/s/article/PAS-backup-prepare-job-renamed-to-backup-restore-may-cause-automation-failures)
-		* [Buildpacks are missing or not installed in PAS 2.3](https://community.pivotal.io/s/article/buildpacks-are-missing-or-not-installed-in-pas-2-3)
-		* [Diego Cell garden healthcheck fails and becomes unhealthy](https://community.pivotal.io/s/article/diego-cell-garden-healthcheck-fails-and-becomes-unhealthy)
+        * [PAS backup-prepare job renamed to backup-restore may cause automation failures](https://community.pivotal.io/s/article/PAS-backup-prepare-job-renamed-to-backup-restore-may-cause-automation-failures)
```

Comment:
This indentation looks a bit different in the diff, but as far as I can tell the markdown renders correctly.